stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/viewmodel/channel/ChannelListViewModel.kt

```diff
@@ -250,7 +254,19 @@ public class ChannelListViewModel(
         }
 
         channelsState = channelsState.copy(isLoadingMore = true)
-        chatDomain.queryChannelsLoadMore(filter, currentConfig.querySort).enqueue()
+        if (ToggleService.isEnabled(ToggleService.TOGGLE_KEY_OFFLINE)) {
+            loadMoreChannelsWithOfflinePlugin(filter, currentConfig.querySort)
```

Comment:
Nit: We can builtin invocations and remove these methods, since they're one-liners